asreview/webapp/start_flask.py

```diff
@@ -31,15 +33,24 @@
 from asreview.project import ASReviewProject
 from asreview.project import get_project_path
 from asreview.project import list_asreview_projects
-from asreview.webapp import api
+from asreview.utils import asreview_path
+from asreview.webapp.api import auth, projects, team
+
+from asreview.webapp import DB
+from asreview.webapp.authentication.models import (
+    SingleUser,
+    User
+)
+from asreview.webapp.authentication.oauth_handler import OAuthHandler
 
 # set logging level
-if os.environ.get('FLASK_ENV', "") == "development":
+if os.environ.get('FLASK_DEBUG', "") == "1":
```

Comment:
Maybe support both version? Then we don't have do bump flask in setup.py.

Reply:
Done